pkg/reconcilers/configmap.go

```diff
@@ -18,3 +19,29 @@ func ConfigMapReconcileField(desired, existing *v1.ConfigMap, fieldName string)
 	}
 	return updated
 }
+
+func RedisConfigMapReconcileField(desired, existing *v1.ConfigMap, fieldName string) bool {
+	updated := false
+
+	if existingVal, ok := existing.Data[fieldName]; !ok {
+		existing.Data[fieldName] = desired.Data[fieldName]
+		updated = true
+	} else {
+		existingString := existingVal
+
+		replicaOfString := "rename-command REPLICAOF \"\""
+		if !strings.Contains(existingString, replicaOfString) {
+			newString := existingString + "\n" + replicaOfString
```

Comment:
I will leave this up to you. I would not do this. I would not "modify" a file concatenating strings. It's error prone. If the `rename-command REPLICAOF \"\"` is not there, the file is invalid and hence, the operator updates it entirely.

Reply:
the idea was to append to the existing config if not present rather than fully replace it. This allows for a custom configuration to continue. It's not clear from the documentation if edits to the configuration are allowed / supported but we don't prevent a user from doing so, so we can assume at least some customers have done so.

Reply:
LGTM then